evm_arithmetization/src/generation/mod.rs

```diff
@@ -130,6 +132,10 @@ pub struct GenerationInputs<F: RichField> {
     ///
     /// This is specific to `cdk-erigon`.
     pub ger_data: Option<(H256, H256)>,
+
+    /// A table listing each JUMPDESTs reached in each call context under
+    /// associated code hash.
```

Comment:
```suggestion
```
Is this comment telling me anything that hovering over the struct wouldn't?

Reply:
No, but it may be nice to have it in this context as well.